src/components/ProfileInput.jsx

```diff
@@ -0,0 +1,109 @@
+import Select from '@material-ui/core/Select';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 300,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ProfileInput() {
+  const classes = useStyles();
+  const [grade, setGrade] = React.useState('');
+  const [gradYear, setGradeYear] = React.useState('');
+  const [major, setMajor] = React.useState('');
+  const [gender, setGender] = React.useState('');
+  const [daca, setDaca] = React.useState('');
+
+  const handleGradeChange = (event) => {
+    setGrade(event.target.value);
+  };
+  const handleGradChange = (event) => {
+    setGradeYear(event.target.value);
+  };
+  const handleMajorChange = (event) => {
+    setMajor(event.target.value);
+  };
+  const handleGenderChange = (event) => {
+    setGender(event.target.value);
+  };
+  const handleDacaChange = (event) => {
+    setDaca(event.target.value);
+  };
+
+  return (
+    <div>
+      <FormControl className={classes.formControl}>
+        <InputLabel>Grade</InputLabel>
+        <Select labelId="demo-simple-select-label" id="demo-simple-select" value={grade} onChange={handleGradeChange}>
+          <MenuItem value={20}>Highschool Senior</MenuItem>
+          <MenuItem value={30}>College Freshman</MenuItem>
+          <MenuItem value={40}>College Sophomore</MenuItem>
+          <MenuItem value={50}>College Junior</MenuItem>
+          <MenuItem value={60}>College Senior</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Graduation Year</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={gradYear}
+          onChange={handleGradChange}
+        >
+          <MenuItem value={10}>2020</MenuItem>
+          <MenuItem value={20}>2021</MenuItem>
+          <MenuItem value={30}>2022</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Major</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={major}
+          onChange={handleMajorChange}
+        >
+          <MenuItem value={10}>Computer Science</MenuItem>
+          <MenuItem value={20}>Chemical Engineering</MenuItem>
+          <MenuItem value={30}>Data Science</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Gender</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
```

Comment:
Btw, multiple HTML elements shouldn't have the same [HTML id attribute](https://www.w3schools.com/html/html_id.asp)

Reply:
Oh ok I see, this is for editing purposes on .CSS file but we are not styling using .CSS so I will simply delete the HTML `labelId` and `id` attributes